lib/dotcom_web/views/helpers.ex

```diff
@@ -53,7 +53,13 @@ defmodule DotcomWeb.ViewHelpers do
   def subway_lines, do: @subway_lines
 
   def svg(unknown) do
-    raise ArgumentError, message: "unknown SVG #{unknown}"
+    cond do
+      unknown =~ ~r/massport|logan-express/ ->
+        svg("icon-mode-bus-default.svg")
+
+      true ->
+        raise ArgumentError, message: "unknown SVG #{unknown}"
```

Comment:
I can't think of a time we would want to raise an error instead of returning *some* icon. Worst case should send an "error" to Sentry saying that an SVG isn't found and just return the default.

Reply:
https://hexdocs.pm/sentry/Sentry.html#capture_message/2

Reply:
Yeah. It's not always gonna merit a bus representation! But it's a fine enough fallback. But what if the file for `svg("icon-mode-bus-default.svg")` mysteriously disappears from `/priv`? But am probably overthinking this point...

Reply:
If we want an even safer default I think the only option left would be to return a `""` or something that'll render but not notably impact the HTML.